doc/releases/changelog-0.26.0.md

```diff
@@ -4,27 +4,84 @@
-  executions per optimization step compared to 3 for `qml.SPSAOptimizer`.
-  The additional circuit executions are used to provide a stochastic estimation of a second-order
-  metric tensor, which often helps the optimizer to achieve faster convergence. 
+  The classical-shadow measurement protocol is described in detail in the
+  [classical shadows paper](https://arxiv.org/abs/2002.08953).
+  As part of the support for classical shadows in this release, two new finite-shot and fully-differentiable measurements are available: 
 
-  Use `qml.QNSPSAOptimizer` like you would any other optimizer:
+  - QNodes returning `qml.classical_shadow` will return two entities: 
+
+    + `bits`: 0 or 1 if the 1 or -1 eigenvalue is sampled, respectively
+    + `recipes`: the randomized Pauli measurements that are performed for each qubit, identified as a unique integer: 0 for Pauli X, 1 for Pauli Y, and 2 for Pauli Z.
+    
+    ```python
+    dev = qml.device("default.qubit", wires=2, shots=3)
+
+    @qml.qnode(dev)
+    def circuit():
+        qml.Hadamard(wires=0)
+        qml.CNOT(wires=[0, 1])
+        return qml.classical_shadow(wires=[0, 1])
+    ```
+
+    ```pycon
+    >>> bits, recipes = circuit()
+    >>> bits
+    tensor([[0, 0],
+            [1, 0],
+            [0, 1]], dtype=uint8, requires_grad=True)
+    >>> recipes
+    tensor([[2, 2],
+            [0, 2],
+            [0, 2]], dtype=uint8, requires_grad=True)
+    ```
+
+  - QNodes returning `qml.shadow_expval` yield the expectation value estimation using classical shadows:
+
+    ```python
+    dev = qml.device("default.qubit", wires=range(2), shots=10000)
+
+    @qml.qnode(dev)
+    def circuit(x, H):
+        qml.Hadamard(0)
+        qml.CNOT((0,1))
+        qml.RX(x, wires=0)
+        return qml.shadow_expval(H)
+
+    x = np.array(0.5, requires_grad=True) 
+    H = qml.Hamiltonian(
+            [1., 1.], 
+            [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)]
+        )  
+    ```
+
+    ```pycon
+    >>> circuit(x, H), 
+    tensor(1.8486, requires_grad=True) 
+    >>> qml.grad(circuit)(x, H))
+    -0.4797000000000001
+    ```
+
+  Fully-differentiable QNode transforms for both new classical-shadows measurements are also available via
+  `qml.shadows.shadow_state` and `qml.shadows.shadow_expval`, respectively.
   
-  ```python
-  max_iterations = 50
-  opt = qml.QNSPSAOptimizer() 
+  For convenient post-processing, we've also added the ability to calculate general Renyi entropies 
+  by way of the `ClassicalShadow` class' `entropy` method, which requires the wires of the subsystem of interest
+  and the Renyi entropy order:
 
-  for _ in range(max_iterations):
-      params, cost = opt.step_and_cost(cost, params)
-  ```  
+  ```pycon
+  >>> shadow = ClassicalShadow(bits, recipes)
+  >>> vN_entropy = shadow.entropy(wires=[0, 3], alpha=1)
+  ``` 
 
 <h4>Qutrits: quantum circuits for tertiary degrees of freedom ☘️</h4>
```

Comment:
I can't comment on it, but:

- We can remove the `type(dev)` line (and the print statement after it)
- In standard paragraphs/text, `default.qutrit` doesn't need the quotation marks :)
- The size of the example can be reduced, by:
  - Eliminating one of the qnodes (only one is needed really)
  - Removing the separate `qutrit_function` and simply including the operations in the QNode
  - removing the import statements